packages/whenable/README.md

```diff
@@ -0,0 +1,55 @@
+# Whenables
+
+Native promises are not compatible with `@agoric/store`, which means that on the Agoric platform, such promises are disconnected when their creator vat is upgraded.  Whenables are storable objects that represent a promise that tolerates disconnections.
+
+## Whenable Consumer
+
+If your vat is a consumer of promises that are unexpectedly fulfilling to a Whenable (something like):
+
+```js
+await w;
+Object [Whenable] {
+  payload: { whenableV0: Object [Alleged: WhenableInternalsKit whenableV0] {} }
+}
+```
```

Comment:
Line 11 through 13 are not JS syntax. They are only Node console output syntax. But there is nothing here to indicate that, leading to reader confusion. (Indeed, even though I know this console output syntax, seeing it like this I was confused for a bit.)

I have no concrete suggestion about how to make this clear, but I do think it needs to be fixed. Do we have precedent in other docs?

Reply:
I see it is fixed. Resolving.